synapse/tests/test_neuron.py

```diff
@@ -396,6 +396,7 @@ def onreg(mesg):
 
                         ok, info = sess.call(('cell:get', {'name': 'cell00@localhost'}))
                         self.true(ok)
+                        self.eq(info['type'], 'synapse.lib.cell.Cell')
```

Comment:
In ``test_neuron_neuron`` lets also spin up a ``synapse.tests.test_neuron.TstCell`` which is a subclass of ``synapse.lib.cell.Cell`` and ensure his cellinfo is ``synapse.tests.test_neuron.TstCell``.